app/styles/app.scss

```diff
@@ -1,5 +1,7 @@
 @import 'variables';
 @import 'ember-paper';
+@import "ember-paper-expansion-panel";
```

Comment:
Where'd this come from? Are there other things we can import (like table styles)?